### PR TITLE
File versions update for release 201908.00

### PR DESCRIPTION
**Pull request by leegeth**
File versions update

Description
-----------
File versions update for release 201908.00

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: libraries/freertos_plus/standard/pkcs11/src/iot_pkcs11.c [hunk 1]
```diff
@@ -1,3 +1,28 @@
+/*
+ * Amazon FreeRTOS PKCS#11 V1.0.0
```
Comment (sarenameas) [inline libraries/freertos_plus/standard/pkcs11/src/iot_pkcs11.c]: Is this supposed to be V2.0.0?

File: libraries/freertos_plus/standard/pkcs11/include/iot_pkcs11.h [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
- * Amazon FreeRTOS
- * Copyright (C) 2017 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ * Amazon FreeRTOS PKCS#11 V1.0.0
```
Comment (sarenameas) [inline libraries/freertos_plus/standard/pkcs11/include/iot_pkcs11.h]: This is supposed to be PKCS #11 (space in between, this was a comment I got in 2017).

Comment (sarenameas) [reply libraries/freertos_plus/standard/pkcs11/include/iot_pkcs11.h]: Also I think it is at V2.0.0?

File: libraries/freertos_plus/standard/pkcs11/src/iot_pkcs11.c (hunk 1 above)
Comment (sarenameas) [reply libraries/freertos_plus/standard/pkcs11/src/iot_pkcs11.c]: `PKCS #11`

Comment (leegeth) [reply libraries/freertos_plus/standard/pkcs11/src/iot_pkcs11.c]: libraries/freertos_plus/standard/pkcs11 is 1.0.0 and 

libraries/abstractions/pkcs11 is 2.0.0

File: libraries/abstractions/pkcs11/include/iot_pkcs11_pal.h [hunk 3]
```diff
@@ -1,3 +1,28 @@
+/*
+ * Amazon FreeRTOS PKCS#11 V2.0.0
```
Comment (sarenameas) [inline libraries/abstractions/pkcs11/include/iot_pkcs11_pal.h]: PKCS #11 with a space

File: libraries/freertos_plus/standard/pkcs11/include/iot_pkcs11.h (hunk 2 above)
Comment (leegeth) [reply libraries/freertos_plus/standard/pkcs11/include/iot_pkcs11.h]: Yea. I followed the same pattern in the 

libraries/abstractions/pkcs11 folder for versioning. This was done by Alexa. Do you think both those needs modification?

File: libraries/abstractions/pkcs11/include/iot_pkcs11_pal.h (hunk 3 above)
Comment (leegeth) [reply libraries/abstractions/pkcs11/include/iot_pkcs11_pal.h]: same as above

File: libraries/freertos_plus/standard/pkcs11/include/iot_pkcs11.h (hunk 2 above)
Comment (sarenameas) [reply libraries/freertos_plus/standard/pkcs11/include/iot_pkcs11.h]: yea those are all wrong lol

**Pull request merged**